code/modules/antagonists/ninja/ninjaDrainAct.dm

```diff
@@ -296,8 +295,23 @@
 	modelselected["Medical"] = "/obj/item/robot_model/ninja/ninja_medical"
 	modelselected["Saboteur"] = "/obj/item/robot_model/ninja_saboteur"
 	//SKYRAT EDIT: ADDITION END
+
 	if(!do_after(ninja, 6 SECONDS, target = src))
 		return
+	//ZUBBER EDIT START - Puts a cap on ninja's borg hacking
+	//Set to negative numbers for infinite uses
+	var/datum/antagonist/ninja/borg_hacks_left = -1
+	if(borg_hacks_left == 0)
+		to_chat(src, span_danger("UPLOAD ABORTED. SUSPICIOUS ACTIVITY DETECTED."))
+		to_chat(ninja, span_danger("UPLOAD FAILURE. TOKEN DENIED."))
+		return
+	else if(borg_hacks_left > 0)
+		borg_hacks_left -= 1
```

Comment:
```suggestion
		borg_hacks_left--
```

Reply:
> 

does it not be a numeric value in cases where the var is set to a number?